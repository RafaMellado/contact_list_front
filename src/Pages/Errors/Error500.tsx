export function Error500() {
  return <h1>Error 500</h1>;
}
