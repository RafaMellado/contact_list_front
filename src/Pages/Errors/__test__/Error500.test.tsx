import { Error500 } from "../Error500";
import { act, render } from "@testing-library/react";

describe(`<${Error500.name} />`, () => {
  const factoryComponent = () => render(<Error500 />);

  test("renders correctly the component", () => {
    const component = factoryComponent();

    act(() => {
      expect(component.asFragment()).toMatchSnapshot();
    });
  });
});
