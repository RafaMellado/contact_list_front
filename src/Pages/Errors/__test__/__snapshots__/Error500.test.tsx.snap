// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error500 /> renders correctly the component 1`] = `
<DocumentFragment>
  <h1>
    Error 500
  </h1>
</DocumentFragment>
`;
